src/test/java/com/parasoft/demoapp/graphql/CartItemGraphQLDataFetcherTest.java

```diff
@@ -73,7 +77,11 @@ public void setup() {
     @Before
     public void conditionalBefore() {
         Set<String> testNames = new HashSet<>(Arrays.asList("test_getCartItems_normal", "test_addItemInCart_normal",
-                "test_removeCartItem_normal", "test_removeAllCartItems_normal"));
+                "test_removeCartItem_normal", "test_removeAllCartItems_normal",
+                "test_updateItemInCart_normal",
+                "test_updateItemInCart_invalidItemQty_equalsZero",
```

Comment:
Why do we need to include these tests from line 82 to 83? If they are assumed to be failed, I assume the test should not make successful changes to database.

Reply:
If we want to update cart items, we need add items into cart first, so these tests change the database.